### ISSUE TITLE
Option to turn off color output

### ISSUE THREAD

**Issue by awshew** [kovetskiy/mark#69]
Please provide an option to turn off ANSI color output - our pipeline tool (ConcourseCI) doesn't interpret the 256 ANSI code outputted by mark properly, and results in blinking text which is nearly impossible to read. A bug was raised with them and auto-closed - https://github.com/concourse/concourse/issues/2390

**Comment by kovetskiy** [comment]
Hi, I'm not actually sure why you would need an option to disable it, because mark automatically detects that the stdin is not a tty. So if you run mark in a script, it should not show a colored output, you can easily confirm it by something like that:

```
 $  mark -u e -p p -b b -f test.md --dry-run 
 $  mark -u e -p p -b b -f test.md --dry-run 2>&1 | cat
```

Does concourse somehow emulate tty mode?

![image](https://user-images.githubusercontent.com/8445924/111416333-c7f81a80-86f4-11eb-95cc-7f5a8c7d5409.png)

**Comment by awshew** [comment]
Concourse doesn't process the ANSI codes mark uses properly, rendering all output text as a blinking blob. While the fix for that lies with them, this wasn't an issue with mark prior to 4.0 so we'd like to skip colour altogether if possible.

**Comment by kovetskiy** [comment]
The option is available in version 5.4, just pass `--color never` and mark will not colorize the output.
